models/data_warehouse_l.model.lkml

```diff
@@ -69,7 +19,7 @@ explore: user_events_telemetry {
 
   join: server_fact {
     view_label: "Server Fact"
-    sql_on: ${license_server_fact.server_id} = ${server_fact.server_id} ;;
+    sql_on: COALESCE(${license_server_fact.server_id},${license_server_fact2.server_id}) = ${server_fact.server_id} ;;
```

Comment:
I tried using an OR condition here, but it was doing a cartesian join, `COALESCE` fixes this and eliminates cartesian join.